action.yml

```diff
@@ -117,9 +157,9 @@ runs:
       shell: bash
       run: |
           if [[ "${{ github.event_name }}" == "issue_comment" ]]; then
-            echo "::set-output name=artifact::plans-${{ github.event.issue.number }}"
+            echo "artifact=plans-${{ github.event.issue.number }}" >> $GITHUB_OUTPUT
```

Comment:
fix deprecation warnings

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/